src/Style/Sass/main.scss

```diff
@@ -30,4 +30,5 @@
 @import "pages/create";
 @import "pages/auth";
 @import "layout/termsAndConditions";
-@import "themes/theme1";
\ No newline at end of file
+@import "themes/theme1";
+@import "layout/articleSocialSharing";
```

Comment:
please avoid using long names @murediane

Reply:
which name can you suggest here?